srcDLL/DllMain.cpp

```diff
@@ -29,10 +29,10 @@ void OnShutdown();
 // Globals from other files should not use these before DllMain has started
 // Similarly these should not use globals from other files before DllMain has started
 // Pay careful attention to anything passed to a constructor, or called by a constructor
-LoggerFile loggerFile; // Logging to file in Outpost 2 folder
-LoggerMessageBox loggerMessageBox; // Logging to pop-up message box
-LoggerDistributor loggerDistributor({&loggerFile, &loggerMessageBox});
-LoggerDebug loggerDebug;
+LogFile logFile; // Logging to file in Outpost 2 folder
+LogMessageBox logMessageBox; // Logging to pop-up message box
+LogDistributor logDistributor({&logFile, &logMessageBox});
+LogDestinationDebug logDestinationDebug;
```

Comment:
Why does this one have "Destination" in the name? It seems inconsistent. Accidental search/replace?

Reply:
LogDebug already exists as a global level function. I wasn't sure we wanted global function and a class with the same signature. Maybe there is an alternate name like LogDebugMessage that would be better?